Apps/WebClient/src/ClientApp/src/store/modules/notification/types.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    ActionContext,
+    ActionTree,
+    GetterTree,
+    Module,
+    MutationTree,
+} from "vuex";
+
+import { ErrorType } from "@/constants/errorType";
+import ApiResult from "@/models/apiResult";
+import { ResultError } from "@/models/errors";
+import Notification from "@/models/notification";
+import { LoadStatus, Operation } from "@/models/storeOperations";
+import { RootState } from "@/store/types";
+
+export interface NotificationState {
+    notifications: Notification[];
+    statusMessage: string;
+    error?: ResultError;
+    status: LoadStatus;
+    lastOperation: Operation | null;
+}
+
+export interface NotificationGetters
+    extends GetterTree<NotificationState, RootState> {
+    notifications(state: NotificationState): Notification[];
+}
+
+type StoreContext = ActionContext<NotificationState, RootState>;
+export interface NotificationActions
+    extends ActionTree<NotificationState, RootState> {
+    retrieve(
+        context: StoreContext,
+        params: { hdid: string }
+    ): Promise<ApiResult<Notification[]>>;
+    deleteNotification(
+        context: StoreContext,
+        params: { hdid: string; notification: Notification }
+    ): Promise<void>;
+    deleteNotifications(
```

Comment:
Corresponding backend code has been named dismiss.  Was there a reason to not keep things consistent.